components/script/dom/window.rs

```diff
@@ -1984,6 +1986,23 @@ impl Window {
             return;
         }
 
+        // Send resize message to any local `Pipeline`s synchronously, so that the value
```

Comment:
"synchronously" makes it sound like you are blocking on the sending of messages, so I think it's more precise to write that you are batching those messages locally for them to be handled at the next update the rendering(which is better than writing "layout", because the concept is more clearly defined across layout and script boundaries thanks to the html spec).

Reply:
Okay. I've reworded this a bit to be clearer.